RemoteSpiNNaker/RemoteSpiNNakerWeb/src/main/java/uk/ac/manchester/cs/spinnaker/rest/utils/RestClientUtils.java

```diff
@@ -88,9 +90,24 @@ private RestClientUtils() {
      * @return the client
      */
     protected static ResteasyClient createRestClient(final URL url) {
+
         try {
             SSLContextBuilder builder = new SSLContextBuilder();
-            builder.loadTrustMaterial(null, new TrustSelfSignedStrategy());
+            builder.loadTrustMaterial(new TrustSelfSignedStrategy());
+            String trustStore = System.getProperty(
```

Comment:
It might be better to factor out the loading of the trust store somehow.

Reply:
You mean move that to a new function?

Reply:
Done